psh/test-control.py

```diff
@@ -0,0 +1,91 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh control codes test
+#
+# Copyright 2023 Phoenix Systems
+# Author: Damian Modzelewski
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+import tools.psh as psh
+
+
+def CC_parser(harness, input: str, output: str):
+    harness.write(input)
+    if not output:
+        harness.expect(psh.PROMPT)
+    else:
+        harness.expect(f"psh: {output} not found", timeout=-1)
+
+
+"""
+CONTROL CODE CHEAT-SHEAT
+
+(\177 or \033[3~) -> Delete the last character
+\001 -> Move to the first position in the text
+\003 -> Cancel command
+\003[A -> In system declaration of Up arrow or 8 on numeric keypad
```

Comment:
There is not test for this code ?

Reply:
It is tested in line 54. There is a typo, suppose to be `\033[A` instead of `\003[A`